CMakeLists.txt

```diff
@@ -1261,8 +1261,13 @@ macro(add_test_prog prog)
                           ${LIB_PLATFORM}
                           event_core
                           event_extra
+                          event_pthreads
```

Comment:
Better to pass event_pthreads only for the new example, as you can see here we ass ARGN so it will work (take a look at how m is passed for one exmaple)

Reply:
Oh, sorry for that. it's not supposed to remain here, it's a legacy line.